projects/02-meme-generator-api/implementation/app/models/meme.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'mini_magick'
+require 'open-uri'
+
+class Meme
+  attr_accessor :image_url, :text, :file_name
+
+  def create
+    save_image
+    add_text
+  end
+
+  def self.file_path(file_name)
+    "app/images/#{file_name}"
+  end
+
+  private
+
+  def save_image
+    uri = URI.parse(image_url)
+    @file_name = 'image1.jpg'
+    @meme_path = "app/images/#{@file_name}"
```

Comment:
I guess we can reuse the class method here instead of duplicating the path generation logic

Instead of 
```
    @meme_path = "app/images/#{@file_name}"
```

call the class method as:

```
    @meme_path = self.class.file_path(@file_name)
```